publisher-utils/utils/visualize_publishers.py

```diff
@@ -0,0 +1,55 @@
+import asyncio
+import time
+
+import matplotlib.pyplot as plt
+import pandas as pd
+from pontis.core.client import PontisClient
+from pontis.core.utils import currency_pair_to_key, felt_to_str
+
+
+async def main(pair):
+    key = currency_pair_to_key(*pair)
+
+    client = PontisClient()
+    decimals = await client.get_decimals(key)
+    entries = []
+
+    num_reps = 12 * 3  # Run for 3 hours
```

Comment:
Looks very good, some of this code can be refactored for precision on timing (instead of 36 to do 5minutes for 3 hours, have it directly possible to describe the increment an then the total time one line or function call lines 17-24), but general W on getting this to csv conversion. 

Can make this a CSV util for an API and rename file later.

Reply:
Good point, I like the idea! I think this file will change a lot and actually migrate to a React frontend, but wanted to get this MVP up.